core/common/src/main/java/alluxio/resource/ResourcePool.java

```diff
@@ -102,15 +102,14 @@ public T acquire(Integer time, TimeUnit unit) {
       return createNewResource();
     }
 
-    mCurrentCapacity.decrementAndGet();
```

Comment:
`mCurrentCapacity` represents the total number of objects that have ever been created for this pool. Calling `mCurrentCapacity.getAndIncrement()` on line 100 acquires permission to create a new resource for the pool. However, if incrementing the capacity would move use over the max capacity, we cannot create a new resource, so we release the permission to create a new resource by calling `mCurrentCapacity.decremenetAndGet()`.